src/Entity/Company.php

```diff
@@ -72,6 +72,13 @@ public static function createFromRegistration(CompanyRegistration $registration)
         ;
     }
 
+    public static function createFromPayload($username, array $payload)
```

Comment:
Please, add type hint

Reply:
Can't do that. The implementation must be compatible with the signature of:
```
Lexik\Bundle\JWTAuthenticationBundle\Security\User\JWTUserInterface::createFromPayload($username, array $payload)
```